### PR TITLE
BSRD-263: Improve man

### PR DESCRIPTION
**Pull request by Tonow-c2c**
Related to [https://jira.camptocamp.com/browse/BSRD-263](https://jira.camptocamp.com/browse/BSRD-263)

### CODE REVIEW

**Review by vrenaville — APPROVED** [review]

File: jira_projects_create_cards.py [hunk 1]
```diff
@@ -310,13 +310,26 @@ def main(
...
         csv_jira_card):
     """
-        Create a Jira issue on project list retrieved in GitHub
-
-        The project list is retrieved from GitHub repositories
-        search using topic filter.
-        Multiple topics can be combined like "odoo-9,need-5-digits".
-        If you don't complete directly the CLI option, it is not a problem,
-        you will be prompted to use this tool.
+        Create a Jira issues on project list.
```
Comment (asaunier) [inline jira_projects_create_cards.py]:
```suggestion
        Create a Jira issue on project list.
```

File: jira_projects_create_cards.py [hunk 2]
```diff
@@ -310,13 +310,26 @@ def main(
...
+
+        1) Project list :
+        * The list can be directly give by csv file (projects.csv) in argument
```
Comment (asaunier) [inline jira_projects_create_cards.py]:
```suggestion
        * The list can be directly provided by csv file (projects.csv) in argument
```
or at least "giveN"

File: jira_projects_create_cards.py [hunk 3]
```diff
@@ -310,13 +310,26 @@ def main(
...
+            --csv_project_list projects.csv
+        * Or retrieved in GitHub by corresponding label
+            --topic business,odoo-9,need-5-digits
+
+        And the list can be edited in with before card creation in the shell
```
Comment (asaunier) [inline jira_projects_create_cards.py]:
```suggestion
        And the list can be edited before card creation in the shell
```

File: jira_projects_create_cards.py [hunk 4]
```diff
@@ -310,13 +310,26 @@ def main(
...
+        interaction
+
+        2) Card content :
+        * The contant can be dublicate from existing card
```
Comment (asaunier) [inline jira_projects_create_cards.py]:
```suggestion
        * The content can be duplicated from an existing card
```

File: jira_projects_create_cards.py [hunk 5]
```diff
@@ -310,13 +310,26 @@ def main(
...
+        1) Project list :
+        * The list can be directly give by csv file (projects.csv) in argument
+            --csv_project_list projects.csv
+        * Or retrieved in GitHub by corresponding label
+            --topic business,odoo-9,need-5-digits
+
+        And the list can be edited in with before card creation in the shell
+        interaction
+
+        2) Card content :
```
Comment (asaunier) [inline jira_projects_create_cards.py]:
```suggestion
        2) Card content:
```

File: jira_projects_create_cards.py [hunk 6]
```diff
@@ -310,13 +310,26 @@ def main(
...
+        * The contant can be dublicate from existing card
+            --template JRA-1330
+        * Or give by csv file (card_template.csv) in argument
```
Comment (asaunier) [inline jira_projects_create_cards.py]:
```suggestion
        * Or provided by csv file (card_template.csv) in argument
```

File: jira_projects_create_cards.py [hunk 7]
```diff
@@ -310,13 +310,26 @@ def main(
...
+            --csv_jira_card card_template.csv
+        * Or give directly in arguments
```
Comment (asaunier) [inline jira_projects_create_cards.py]:
```suggestion
        * Or provided directly in arguments
```

**Pull request merged**